apps/shell/app.js

```diff
@@ -8,6 +8,8 @@ var hbs       = require('hbs');
 var dotenv    = require('dotenv');
 var Tokens    = require('csrf');
 var url       = require('url');
+var yaml      = require('js-yaml');
+var glob      = require("glob");
 var port      = 3000;
```

Comment:
Make all of these constants.

```suggestion
const dotenv    = require('dotenv');
const Tokens    = require('csrf');
const url       = require('url');
const yaml      = require('js-yaml');
const glob      = require("glob");
const port      = 3000;
```